OneCommunityGlobal/HighestGoodNetworkApp
# Ramakrishna fix fetch user history error

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/64008831/847bc893-151b-4153-9409-057389d8c2bd)

## Related PRS (if any):
Please use latest development branch in backend
…

## Main changes explained:
- added proper condition for api call for setupinvitationdata in setupinviation component
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as any user(volunteer preferred) who has permission to see/create users (you can give any user permission using an admin/owner account )
5. Navigate dashboard -> user Management 
6. And absorb you didn't see any fetching Error: Invitation History toast message.
7. Try to repeat the above two steps(5,6) to another usertypes(like mentor, manager) who has permission to see/create users.
